parts/linux/cloud-init/artifacts/components.json

```diff
@@ -27,8 +27,18 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/coredns",
-          "latestVersion": "v1.9.4-hotfix.20240704",
-          "previousLatestVersion": "v1.9.4-hotfix.20240627"
+          "latestVersion": "v1.9.4-hotfix.20240704"
+        }
+      ]
+    },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/v2/kubernetes/coredns:*",
+      "amd64OnlyVersions": [],
+      "multiArchVersionsV2": [
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/kubernetes/coredns",
+          "latestVersion": "v1.11.3-2",
+          "previousLatestVersion": "v1.9.4-2"
```

Comment:
latest/previousLatest should be with respect to the particular minor version - in you're case since you are adding a single new patch for 1.11, I'd just omit previousLatestVersion altogether

Reply:
@Devinwong correct me if I'm wrong

Reply:
ya Cameron is right. 
```
{
          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/kubernetes/coredns",
          "latestVersion": "v1.11.3-2"
},
{
          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/kubernetes/coredns",
          "latestVersion": "v1.9.4-2"
}
```
Though both will work.
@SriHarsha001  can I add you as the PR assignee of this component? Feel free to give my other folks' github IDs too.
https://github.com/Azure/AgentBaker/blob/master/.github/README-RENOVATE.md#what-is-the-responsibility-of-a-pr-assignee

Reply:
yes sure, you an add me as the PR assignee for coredns component. I have also made the suggested changes.